### PR TITLE
Removed collective.indexing dependency

### PR DESCRIPTION
**Pull request by agitator**
is there a way to define collective.indexing for Plone versions smaller than 5.1?

### CODE REVIEW

**Comment by agitator** [comment]
@vincentfretin any idea why the tests are failing in https://travis-ci.org/collective/dexterity.membrane/jobs/265935501 or what I could try to fix them?

**Comment by hvelarde** [comment]
check this thread:

https://community.plone.org/t/strange-catalog-behavior-on-plone-5-1/4582?u=hvelarde

you may be facing the same issue.

**Comment by vincentfretin** [comment]
I really don't know. Does the tests passed before those changes?

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +1,38 @@
 language: python
-python: 2.7
-sudo: false
+sudo: true
+dist: xenial
```
Comment (jensens) [inline .travis.yml]: Do we need an explicit dist?

File: .travis.yml [hunk 2]
```diff
@@ -1,23 +1,38 @@
...
+  - PLONE_VERSION=5.1
+before_install:
+#  - sudo apt-get update -qq
+#  - sudo apt-get install -qq python2.7 python2.7-virtualenv
+  - mkdir -p $HOME/buildout-cache/{eggs,downloads}
+  - mkdir $HOME/.buildout
+  - echo "[buildout]" > $HOME/.buildout/default.cfg
+  - echo "download-cache = $HOME/buildout-cache/downloads" >> $HOME/.buildout/default.cfg
+  - echo "eggs-directory = $HOME/buildout-cache/eggs" >> $HOME/.buildout/default.cfg
+  - python --version
```
Comment (jensens) [inline .travis.yml]: outdated way for caching. see the latest discussions at bobtemplates.plone

File: .travis.yml [hunk 3]
```diff
@@ -1,23 +1,38 @@
...
+  - virtualenv --version
+  - pip install --upgrade virtualenv
+  - ./bootstrap-travis.sh
```
Comment (jensens) [inline .travis.yml]: I think its more transparent if you just copy over the commands to ``.travis.yml``.

**Comment by jensens** [comment]
we have some hickup with commits not showing up here, reopen from a different branch

**Comment by jensens** [comment]
which is very odd btw... the new one has those commits in the PR showing up

**Pull request closed without merging**